SetupDataPkg/Tools/VariableList.py

```diff
@@ -1369,6 +1420,26 @@ def main():
             sys.stderr.write('Invalid number of arguments.\n')
             sys.exit(1)
             return
+            
+    if sys.argv[1].lower() == "write_csv_detailed":
+        if len(sys.argv) == 4:
+            schema_path = sys.argv[2]
+            csv_path = sys.argv[3]
+            # Load the schema

```

Comment:
it would be good to validate that schema_path and csv_path are valid (i.e. that they exist)

Reply:
write_csv() was my starting point.
Are you suggesting something like this?
   if not os.path.isfile(schema_path):
        print(f"File does not exist at path: {schema_path}")
        return False
   if not os.path.isfile(csv_path):
        print(f"File does not exist at path: {csv_path}")
        return False